poap-xyz/poap-gallery
# Feature: Added google analytics tracking (PCD-785)

## Summary

Added google analytics tracking to all app

## Type of change


## Deployment guidelines

Add environment variable only to production

```
REACT_APP_GA_MEASUREMENT_ID=G-W21W3VZBBN
```

## Testing

1. Access to the app, and navigate through different routes
2. Make sure this message is not displayed on the browser console: `Tracking not enabled, as trackingId was not given and there is no REACT_APP_GA_MEASUREMENT_ID.`
![image](https://user-images.githubusercontent.com/1110934/143885947-db826cba-7deb-4062-a644-1fe2b8eccd44.png)
3. Go to the debug section in Google Analytics to check the events: https://analytics.google.com/analytics/web/#/m/p296593331/debugview/overview?params=_u..nav%3Dmaui%26_u..pageSize%3D25&collectionId=life-cycle

## Ticket

[PCD-785](https://poap-devs.atlassian.net/browse/PCD-785)

## Dependencies

None

## Checklist:
